clearwater-snmpd/usr/share/clearwater/infrastructure/scripts/snmpd_mem_handler

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#
+# Copyright (C) Metaswitch Networks 2018
+# If license terms are provided to you in a COPYING file in the root directory
+# of the source code repository by which you are accessing this code, then
+# the license outlined in that COPYING file applies to your use.
+# Otherwise no rights are granted except for those provided to you by
+# Metaswitch Networks in a separate written agreement.
+#
+# Called by snmpd to offload requests for the Metaswitch memory OIDs.
+# Syntax:
+# snmp_mem_handler [-n|-g|-s] [oid]
+#   -g - perform a get on the OID
+#   -n - perform a getnext on the OID
+#   -s - perform a set on the OID (rejected by this script)
+#
+# Handles SNMP requests offloaded by snmpd for the Metswitch memory OIDs
+# Handles three different types of request - set, gets and get nexts and
+# returns the appropriate stat based on the contents of /proc/meminfo,
+# making use of the buffers/cache memory as well as the free memory info.
+#
+# The three OIDs this file returns are:
+# oid_prefix.1 = current free (kB)
```

Comment:
Worth just appending the maths we do? i.e. `current free (kB) (free + cached free + buffer free)